devicetypes/smartthings/smartweather-station-tile.src/smartweather-station-tile.groovy

```diff
@@ -275,7 +261,7 @@ def pollUsingZipCode(String zipCode) {
         def sunsetDate = dtf.parse(obs.sunsetTimeLocal)
 
         def tf = new java.text.SimpleDateFormat("h:mm a")
-        tf.setTimeZone(TimeZone.getTimeZone(loc.ianaTimeZone))
+        tf.setTimeZone(TimeZone.getTimeZone(loc?.ianaTimeZone))
 
```

Comment:
If loc is `null` what will this code do? You added the null check but I'm wondering if we need to skip the whole line if loc is null.